docs/regressions-beir-v1.0.0-climate-fever-splade-distil-cocodenser-medium.md

```diff
@@ -0,0 +1,108 @@
+# Anserini Regressions: BEIR (v1.0.0) &mdash; climate-fever, SPLADE-distil CoCodenser Medium
```

Comment:
Our convention is not to have the mode name in the title, otherwise too long...

Reply:
and the line wraps around, and it's ugly...

Reply:
so, just drop the model name, right? e.g.,
# Anserini Regressions: BEIR (v1.0.0) &mdash; climate-fever